### PR TITLE
Handle the case where no homepage is specified anywhere

### PR DESCRIPTION
**Pull request by malept**
Solves the bug where `homepage` in `package.json` is not specified, nor is there a match for a URL in the string version of the `author` field.

I ran into this while fixing some Electron Forge tests.

I wasn't really interested in committing another fixture just to change one `package.json` field, so I have the test copy the `app-without-asar` fixture to `os.tmpdir()` and modify/operate on that directory.

I also took the liberty of removing references to `rimraf` in `test/installer.js`, since `fs-extra` provides `remove*()` functions.

### CODE REVIEW

File: test/installer.js [hunk 1]
```diff
@@ -63,9 +64,37 @@ describe('module', function () {
...
+
+    before(function (done) {
+      if (fs.existsSync(baseDir)) {
+        fs.removeSync(baseDir)
+      }
+      fs.copySync('test/fixtures/app-without-asar', baseDir)
+      var packageJSONFilename = path.join(baseDir, 'resources', 'app', 'package.json')
+      var packageJSON = JSON.parse(fs.readFileSync(packageJSONFilename))
+      packageJSON.author = 'Test Author'
+      fs.writeFileSync(packageJSONFilename, JSON.stringify(packageJSON))
```
Comment (malept) [inline test/installer.js]: I didn't really want to get into FS callback hell, but if you think it's necessary, I can change this. (I'd still keep the `fs.existsSync `block, though).

Comment (jacobq) [inline test/installer.js]: Wouldn't using `require(packageJSONFilename)` also work / be preferable?

Comment (malept) [inline test/installer.js]: Yeah. I still find that a very odd thing that you can do with `require()`, though.

File: src/installer.js [hunk 2]
```diff
@@ -91,6 +91,24 @@ var readLicense = function (options, callback) {
...
 
 /**
+ * Determine the homepage based on the settings in `package.json`.
+ */
+var getHomepage = function (pkg) {
+  if (pkg.homepage) {
+    return pkg.homepage
+  } else if (pkg.author) {
+    if (typeof pkg.author === 'string') {
+      var homepageRegexp = /.*\(([^)]+)\).*/
```
Comment (jacobq) [inline src/installer.js]: Not a big deal, but FWIW, this could also be accomplished by using [`parse-author`](https://github.com/jonschlinkert/parse-author), which might help avoid having to duplicate parsing/regex functionality here.

Comment (malept) [inline src/installer.js]: Sure. I'd rather do that in a separate PR though.
